clang/test/CodeGen/X86/min_max.c

```diff
@@ -0,0 +1,19 @@
+// RUN: %clang_cc1 %s -O2 -triple=x86_64-apple-darwin -emit-llvm -o - | FileCheck %s
```

Comment:
We don't test end-to-end codegen with clang. If you want to test this, you should take the unoptimized clang IR and create a test in the llvm/test/PhaseOrdering directory using it.

Reply:
Out of curiosity, why is end-to-end codegen with clang not used ? Is it too unstable ?